src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/CommonConditions.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.helper;
+
+import org.apache.commons.lang3.StringUtils;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.RespondToOrderData;
+
+import java.util.Map;
+import java.util.Objects;
+
+import static uk.gov.hmcts.reform.finrem.caseorchestration.OrchestrationConstants.YES_VALUE;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.AMENDED_CONSENT_ORDER;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.APPLICANT_REPRESENTED;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.RESP_SOLICITOR_NAME;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.SOLICITOR_AGREE_TO_RECEIVE_EMAILS;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.service.CommonFunction.nullToEmpty;
+
+public class CommonConditions {
```

Comment:
I think this was moved to `CommonFunctions` and `CommonConditions` was completely removed.

@johnoneill46 - haven't you removed it?